user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,65 @@
-# Welcome to the Strapi user documentation!
+# Welcome to the Strapi user guide!
 
-**Strapi is the open-source [Headless CMS](https://strapi.io) developers love.**
\ No newline at end of file
+This user guide contains the functional documentation related to all features available in the admin panel of your Strapi application. 
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development, Staging or Production Environment** <br> When starting working on your application, it is in a development environment, which is the status for the content structuration and application configuration phase. After deploying your application, it is in production or staging. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest<!-- (see Update Strapi version or refer to our migration guides to update your Strapi application) -->. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content managed with Strapi itself, are ruled by a system of permissions. From your Strapi admin panel, you have the possibility to define, at a detailed level, the roles and permissions of all administrators and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
```

Comment:
This is well written but we may want to try and avoid the comma breaks. I tend to use them very often myself and I know my english teacher in high school would yell at me for it :laughing: 

I don't think we need to change anything, just a thought for the future.

Reply:
Thank you for letting me know! I guess that's some French writing habits, we like commas 🙈